src/CMakeLists.txt

```diff
@@ -66,12 +66,14 @@ link_libraries(/usr/lib/x86_64-linux-gnu/libuuid.so)
 link_libraries(/usr/lib/x86_64-linux-gnu/libevent_pthreads.so)
 link_libraries(/usr/lib/x86_64-linux-gnu/libpthread.so)
 link_libraries(/usr/local/lib/libopenvswitch.a) #this was installed by aca-machine-init.sh
+link_libraries(/home/ubuntu/src/github.com/zzxgzgz/marl/build/libmarl.a)
```

Comment:
Please make the corresponding change in machine-init script. 
If we install the library under '/usr/local' then it will be that path.